sdk/main/include/TokenCreateTransaction.h

```diff
@@ -0,0 +1,555 @@
+   */
+  [[nodiscard]] inline std::vector<std::shared_ptr<CustomFee>> getCustomFees() const { return mCustomFees; }
+
+  /**
+   * Get the desired pause key for the new token.
+   *
+   * @return The desired pause key for the new token.
+   */
+  [[nodiscard]] inline std::shared_ptr<Key> getPauseKey() const { return mPauseKey; }
+
+private:
+  /**
+   * Derived from Executable. Construct a Transaction protobuf object from this TokenCreateTransaction object.
+   *
+   * @param client The Client trying to construct this TokenCreateTransaction.
+   * @param node   The Node to which this TokenCreateTransaction will be sent. This is unused.
+   * @return A Transaction protobuf object filled with this TokenCreateTransaction object's data.
+   * @throws UninitializedException If the input client has no operator with which to sign this
+   *                                TokenCreateTransaction.
+   */
+  [[nodiscard]] proto::Transaction makeRequest(const Client& client,
+                                               const std::shared_ptr<internal::Node>& /*node*/) const override;
+
+  /**
+   * Derived from Executable. Submit this TokenCreateTransaction to a Node.
+   *
+   * @param client   The Client submitting this TokenCreateTransaction.
+   * @param deadline The deadline for submitting this TokenCreateTransaction.
+   * @param node     Pointer to the Node to which this TokenCreateTransaction should be submitted.
+   * @param response Pointer to the TransactionResponse protobuf object that gRPC should populate with the response
+   *                 information from the gRPC server.
+   * @return The gRPC status of the submission.
+   */
+  [[nodiscard]] grpc::Status submitRequest(const Client& client,
+                                           const std::chrono::system_clock::time_point& deadline,
+                                           const std::shared_ptr<internal::Node>& node,
+                                           proto::TransactionResponse* response) const override;
+
+  /**
+   * Build a TokenCreateTransactionBody protobuf object from this TokenCreateTransaction object.
+   *
+   * @return A pointer to a TokenCreateTransactionBody protobuf object filled with this TokenCreateTransaction object's
+   *         data.
+   */
+  [[nodiscard]] proto::TokenCreateTransactionBody* build() const;
+
+  /**
+   * The publicly visible name of the token. The token name is specified as a string of UTF-8 characters in Unicode.
+   * UTF-8 encoding of this Unicode cannot contain the 0 byte (NUL). The token name is not unique. Maximum of 100
+   * characters.
+   */
+  std::string mName;
+
+  /**
+   * The publicly visible token symbol. The token symbol is specified as a string of UTF-8 characters in Unicode. UTF-8
+   * encoding of this Unicode cannot contain the 0 byte (NUL). The token symbol is not unique. Maximum of 100
+   * characters.
+   */
+  std::string mSymbol;
```

Comment:
In [Java SDK](https://github.com/hashgraph/hedera-sdk-java) implementation for `TokenCreateTransaction`, this filed is named [tokenSymbol](https://github.com/hashgraph/hedera-sdk-java/blob/274012d54d75c5896414b8406df76a637eb9c58a/sdk/src/main/java/com/hedera/hashgraph/sdk/TokenCreateTransaction.java#L83).
As the [SDK reference](https://github.com/hashgraph/hedera-sdk-reference/issues) documentation is not well specified yet, I would prefer to use the same naming between C++ and Java versions.